live-examples/css-examples/backgrounds-and-borders/border-radius.html

```diff
@@ -0,0 +1,72 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-radius">
+
+	<div class="example-choice" initial-choice="true">
+		<pre><code id="example_one" class="language-css">border-radius: 10px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+			<pre><code id="example_two" class="language-css">border-radius: 10px 5%;</code></pre>
+			<button type="button" class="copy hidden" aria-hidden="true">
+					<span class="visually-hidden">Copy to Clipboard</span>
+			</button>
+	</div>
+
+	<div class="example-choice">
+			<pre><code id="example_three" class="language-css">border-radius: 2px 4px 2px;</code></pre>
+			<button type="button" class="copy hidden" aria-hidden="true">
+					<span class="visually-hidden">Copy to Clipboard</span>
+			</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_four" class="language-css">border-radius: 1px 0 3px 4px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_five" class="language-css">border-radius: 10px 5% / 20px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_six" class="language-css">border-radius: 10px 5% / 20px 30px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_seven" class="language-css">border-radius: 10px 5px 2em / 20px 25px 30%;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_eight" class="language-css">border-radius: 10px 5% / 20px 25em 30px 35em;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
+	<div class="example-choice">
+		<pre><code id="example_nine" class="language-css">border-radius: 100px;</code></pre>
+		<button type="button" class="copy hidden" aria-hidden="true">
+				<span class="visually-hidden">Copy to Clipboard</span>
+		</button>
+	</div>
+
```

Comment:
I think the values chosen here are too subtle and too similar, so the difference between them isn't obvious enough.

Also, we should try to restrict ourselves to 6 choices here, so as to avoid vertical scrollbars. You have 9 here. Choosing 6 representative syntaxes is tricky for this one because the syntax is quite complex. I think we should aim to demonstrate:

* that you can pass a single value to get all 4 rounded corners the same
* that you can pass multiple single-radius values to get different corners with different roundednesses
* that you can pass pairs of radius-values to get elliptical corners
* that you can pass multiple pairs, to get different ellipses in different corners.

I'd probably go for something like:

```
border-radius: 30px;

border-radius: 25% 10%;

border-radius: 10% 30% 50% 70%;

border-radius: 10% / 50%;

border-radius: 20px 100px / 120px;

border-radius: 50% 20% / 10% 40%;
```